Makefile

```
@@ -71,8 +77,8 @@ test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
 .PHONY: e2e
-e2e:
-	go test -v -count=1 -tags e2e ./e2e
+e2e: test-summary-tool ## Runs e2e tests
+	$(GOTESTSUM) --format testname -- -tags=e2e ./e2e
```

```suggestion
	$(GOTESTSUM) --format testname -- -count=1 -tags=e2e ./e2e
```

We need to want to disabling caching to ensure the tests run each time

---

Done